rocketpy/motors/tank.py

```diff
@@ -485,6 +488,13 @@ def underfill_height_exception(param_name, param):
             elif (height < bottom_tolerance).any():
                 underfill_height_exception(name, height)
 
+    def _discretize_fluids(self):
+        """Discretizes the fluid densities according to the flux time and the
```

Comment:
```suggestion
    def _discretize_fluidd_density(self):
        """Discretizes the fluid densities according to the flux time and the
```

Reply:
Much better, thanks for the suggestion, I will implement it.